content/en/continuous_integration/tests/java.md

```diff
@@ -287,10 +280,51 @@ test {
 }
 {{< /code-block >}}
 
-Run your tests as you normally do, specifying the environment where test are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
+{{% /tab %}}
+{{< /tabs >}}
+
+## Running your tests
+
+{{< tabs >}}
+{{% tab "Maven" %}}
+
+Run your tests using the `MAVEN_OPTS` environment variable to specify a path to the Datadog Java Tracer JAR.
+
+In the tracer arguments, specify the following:
+
+* CI Visibility is enabled by setting the `dd.civisibility.enabled` property to `true`.
+* The environment where tests are being run (for example, `local` when running tests on a developer workstation or `ci` when running them on a CI provider) is defined in the `dd.env` property.
+* The name of the service or library under `test` is defined in the `dd.service` property.
```

Comment:
To me highlighting `test` as code here seems slightly confusing.

In this case "library under test" means "library which is being tested". So "test" is used more as a general term, rather than as a specific command (the command itself might be different, for instance with Maven it can be `verify`).

Perhaps we could change the phrasing here then, for example to something like "the name of the service or library that is being tested is defined in the `dd.service` property"?

The same applies to the Gradle section below.

@alai97 please let me know what you think.

Reply:
```suggestion
* The name of the service or library that is being tested is defined in the `dd.service` property.
```